pkg/monitoring/metrics/virt-controller/vmsnapshot.go

```diff
@@ -0,0 +1,42 @@
+package virt_controller
+
+import (
+	"github.com/machadovilaca/operator-observability/pkg/operatormetrics"
+
+	io_prometheus_client "github.com/prometheus/client_model/go"
+	snapshotv1 "kubevirt.io/api/snapshot/v1beta1"
+)
+
+var (
+	vmSnapshotMetrics = []operatormetrics.Metric{
+		VmSnapshotSucceededTimestamp,
+	}
+
+	VmSnapshotSucceededTimestamp = operatormetrics.NewGaugeVec(
+		operatormetrics.MetricOpts{
+			Name: "kubevirt_vmsnapshot_succeeded_timestamp_seconds",
+			Help: "Returns the timestamp of successful virtual machine snapshot.",
+		},
+		[]string{"vm_name", "snapshot_name", "namespace"},
```

Comment:
```suggestion
		[]string{"name", "snapshot_name", "namespace"},
```

that's the usual label name for VM metrics